.github/workflows/ESlint.yaml

```
@@ -0,0 +1,47 @@
+name: ESlint
+on:
+  push:
+    branches:
+      - main
```

Para funcionar com o que temos hoje será necessário deixar como master
```suggestion
      - master
```